src/VirtoCommerce.DemoSolutionFeaturesModule.Data/Models/Catalog/DemoItemEntity.cs

```diff
@@ -4,6 +4,7 @@
 using VirtoCommerce.CatalogModule.Data.Model;
 using VirtoCommerce.DemoSolutionFeaturesModule.Core.Models.Catalog;
 using VirtoCommerce.Platform.Core.Common;
+using demoFeaturesModule = VirtoCommerce.DemoSolutionFeaturesModule.Core;
```

Comment:
Why we need this?

Reply:
Each module has ModuleConstants. It is a meaningless name;